src/settings/TargetProfileDetail.js

```diff
@@ -25,17 +38,71 @@ class TargetProfileDetail extends React.Component {
       path: 'identifier-types?query=id=!{initialValues.externalIdentifierType}',
       records: 'identifierTypes',
     },
+    jobProfiles: {
+      type: 'okapi',
+      records: 'jobProfiles',
+      path: 'data-import-profiles/jobProfiles?limit=5000&query=dataType==("MARC") sortBy name',
```

Comment:
Could you please investigate possible of usage https://github.com/folio-org/ui-inventory/blob/master/src/constants.js#L200 instead of hard code and the same for dataType MARC ?

Reply:
done